Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -591,6 +591,22 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
     ASSERT(m_identifier);
     WEBPAGE_RELEASE_LOG(Loading, "constructor:");
 
+    auto blockIOKit = parameters.store.getBoolValueForKey(WebPreferencesKey::blockIOKitInWebContentSandboxKey())
```

Comment:
let's call this `shouldBlockIOKit` since this is a boolean.

Reply:
Fixed.

Thanks for reviewing!